website/scripts/should-build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# This is run during the website build step to determine if we should skip the build or not.
+# More information: https://vercel.com/docs/platform/projects#ignored-build-step
+
+if [[ "$VERCEL_GIT_COMMIT_REF" == "stable-website"  ]] ; then
```

Comment:
Does this remove the vercel deploys on master as well?

https://packer-git-master-hashicorp.vercel.app has been very useful.

Reply:
It does not! This does a diff against the previous commit to determine if anything in the website directory has changed if we are not on stable-website. This retains current behavior for main, and always deploys on stable-website.

Reply:
Awesome - good stuff.